staging/src/k8s.io/library-go/scheduling/OWNERS

```diff
@@ -0,0 +1,11 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+- sig-scheduling-maintainers
+- sttts
+- luxas
+reviewers:
+- sig-scheduling
+- dixudx
```

Comment:
where is this list of extra people coming from?

Reply:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kube-scheduler/OWNERS

Reply:
Done